### ISSUE TITLE
This code does not work for me (

### ISSUE THREAD

**Issue by chulpanovich** [SukkoPera/NESPlusPlus#1]
Hello. Please help me get this code to work.

**Comment by SukkoPera** [comment]
What's the exact problem?

**Comment by chulpanovich** [comment]
> What's the exact problem?

I just can't get it to work as written in the job description. It does not respond to combinations and the reset button does not work either. Although I tried to connect everything according to the pinout indicated in the sketch.

**Comment by SukkoPera** [comment]
Well, I haven't touched this for the last 3/4 years, but it used to work. I had a quick look at it a few months ago but didn't see anything odd, I'll try again.

**Comment by chulpanovich** [comment]
Please, if it doesn't take you long. I really want to use your development in addition to my new project.

**Comment by monsterro** [comment]
This code seems to be written for a front loader NES, which has active high reset line. I have an AV Famicom and I made some changes to the code to account for its active low reset. I'll make a pull request once I'm done with testing.

**Comment by SukkoPera** [comment]
Yup, we only have Front Loaders in Italy, I've never seen a Top Loader or AV Famicom "in the flesh" and had no idea it had an active-low reset signal.

**Comment by chulpanovich** [comment]
> This code seems to be written for a front loader NES, which has active high reset line. I have an AV Famicom and I made some changes to the code to account for its active low reset. I'll make a pull request once I'm done with testing.

Hello.
Please tell me, did you manage to check the work on the AV Famicom? And also please tell me, does this code from the repository work on the frontal NES?

**Comment by SukkoPera** [comment]
For sure it works on the front-loader, that's the machine it was developed on...

If @monsterro shares the changes he made to get it working on the top-loader/AV, I can integrate them in the code.

**Comment by chulpanovich** [comment]
> For sure it works on the front-loader, that's the machine it was developed on...
> 
> If @monsterro shares the changes he made to get it working on the top-loader/AV, I can integrate them in the code.

Thank you, this will be great)

**Comment by Arnold101X** [comment]
> For sure it works on the front-loader, that's the machine it was developed on...
> 
> If [@monsterro](https://github.com/monsterro) shares the changes he made to get it working on the top-loader/AV, I can integrate them in the code.

ciao ti ho mandato una email ti è arrivata?

**Comment by Arnold101X** [comment]
this works, except for reset, on the famicom or not?

**Comment by Arnold101X** [comment]
succhiaminchia
